pkg/executables/kubectl.go

```diff
@@ -56,13 +57,41 @@ type Kubectl struct {
 }
 
 func (k *Kubectl) SearchCloudStackMachineConfig(ctx context.Context, name string, kubeconfigFile string, namespace string) ([]*v1alpha1.CloudStackMachineConfig, error) {
-	// Required implementation for Kubectl interface implementation
-	return nil, fmt.Errorf("cloudstack provider does not support this yet")
+	params := []string{
+		"get", eksaCloudStackMachineResourceType, "-o", "json", "--kubeconfig",
+		kubeconfigFile, "--namespace", namespace, "--field-selector=metadata.name=" + name,
+	}
+	stdOut, err := k.Execute(ctx, params...)
+	if err != nil {
+		return nil, fmt.Errorf("error searching eksa CloudStackMachineConfigResponse: %v", err)
```

Comment:
Can drop the `error` from all the error messages.

Reply:
done